### PR TITLE
Sommelier v8 Upgrade

### PR DESCRIPTION
**Pull request by cbrit**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new target for running end-to-end tests for validator incentives.
	- Added support for a new addresses module for Cosmos and EVM address mapping.
	- Enhanced the cellarfees module to initiate fee auctions based on USD value.
	- Implemented a new reward distribution mechanism for validators.

- **Bug Fixes**
	- Corrected import paths for the Gravity module from version 4 to version 5 across various files.

- **Documentation**
	- Updated README files to reflect major version 8 changes and new features.

- **Tests**
	- Added comprehensive tests for validator incentives and auction functionalities. 

- **Refactor**
	- Improved parameter management and validation logic within the incentives module. 

- **Chores**
	- Updated dependencies and import paths to maintain compatibility with the latest Gravity module.

### CODE REVIEW

File: app/upgrades/v8/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# v8 upgrade
+
+This upgrade moves Sommelier to major version 8.
+
+## Summary of changes
+
+* Add the addresses module for mapping cosmos/evm addresses
+* Update the cellarfees module to start fee auctions based on the accrued USD value of a particular denom
+* Update the auction module to allow a portion of SOMM proceeds earned by auctions to be burned
+* Upgrade the gravity module to v5
```
Comment (zmanian) [inline app/upgrades/v8/README.md:10]: should mention the new validator incentives

File: x/cellarfees/migrations/v1/keeper/abci.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/peggyjv/sommelier/v8/app/params"
+)
+
+// BeginBlocker emits rewards each block they are available by sending them to the distribution module's fee collector
+// account. Emissions are a constant value based on the last peak supply of distributable fees so that the reward supply
+// will decrease linearly until exhausted.
+func (k Keeper) BeginBlocker(ctx sdk.Context) {
```
Comment (zmanian) [inline x/cellarfees/migrations/v1/keeper/abci.go:12]: I am confused why this is running during a migration.... and what the semantics of this are....

**Comment by zmanian** [comment]
Are there any migrations actually running other than the params migration?

**Comment by cbrit** [comment]
> Are there any migrations actually running other than the params migration in incentives?

x/cellarfees is deleting some old state, has params changes. x/auction also has new params.

**Pull request merged**